packages/analytics-js-integrations/src/integrations/Clevertap/browser.js

```diff
@@ -55,10 +54,7 @@ class Clevertap {
   }
 
   init() {
-    const sourceUrl =
-      document.location.protocol === 'https:'
-        ? 'https://d2r1yp2w7bby2u.cloudfront.net/js/clevertap.min.js'
-        : 'http://static.clevertap.com/js/clevertap.min.js';
+    const sourceUrl = 'https://static.clevertap.com/js/clevertap.min.js';
```

Comment:
any reason for preferring this url over cloudfront.net one ? or both route to same endpoint ?

Reply:
Yes @krishna2020, I checked by hashing both the returned snippets and comparing them, they were identical